### PR TITLE
Include refunded_at and refund_description in ticket cancellation workflow

### PR DESCRIPTION
**Pull request by vidya-ram**
1. Add `refunded_at` and `refund_description` in `process_line_item_cancellation` call
2. Add default message for refunds in cash receipt.

### CODE REVIEW

File: boxoffice/templates/cash_receipt.html [hunk 1]
```diff
@@ -237,7 +237,13 @@ <h1 class="header">Cash Receipt</h1>
         {% for transaction in order.refund_transactions -%}
           <tr>
               <td></td>
-              <td colspan="2">{{ transaction.refund_description }}</td>
+              {%- if transaction.refund_description %}
+                <td colspan="2">{{ transaction.refund_description }}</td>
+              {%- elif transaction.refunded_at %}
+                <td colspan="2">Refund on {{ transaction.refunded_at | invoice_date('%d %b %Y') }}</td>
```
Comment (shreyas-satish) [inline boxoffice/templates/cash_receipt.html]: "Refunded on ...".

Comment (vidya-ram) [inline boxoffice/templates/cash_receipt.html]: Fixed

File: boxoffice/views/order.py [hunk 2]
```diff
@@ -408,7 +409,7 @@ def process_line_item_cancellation(line_item):
         rp_resp = razorpay.refund_payment(payment.pg_paymentid, refund_amount)
         if rp_resp.status_code == 200:
             db.session.add(PaymentTransaction(order=order, transaction_type=TRANSACTION_TYPE.REFUND,
-                online_payment=payment, amount=refund_amount, currency=CURRENCY.INR))
+                online_payment=payment, amount=refund_amount, currency=CURRENCY.INR, refunded_at=func.utcnow(), refund_description='{ticket} cancelled'.format(ticket=line_item.item.title)))
```
Comment (shreyas-satish) [inline boxoffice/views/order.py]: I suggest `refund_description=Refund: {line_item_title}`.

Comment (vidya-ram) [inline boxoffice/views/order.py]: Fixed

**Pull request merged**